.github/workflows/ci.yml

```diff
@@ -169,6 +169,8 @@ jobs:
             - name: Unit Tests
               if: ${{ !cancelled() && steps.setup_success.outcome == 'success' && steps.ts_build.outcome == 'success' }}
               run: yarn test:unit
+              env:
+                  BASE_RPC_URL: ${{ secrets.BASE_RPC_URL }}
```

Comment:
I think you should add BASE_RPC_URL to the top of the file in the `env` section, https://github.com/river-build/river/pull/1727/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR4, and refer to the env var directly here as in 

`BASE_RPC_URL: ${{ env.BASE_RPC_URL }}`.

For some reason foundry.toml is not reading the env var likely because it is not defined at the top level of the job.

Reply:
Still not working. No clue.
https://github.com/river-build/river/actions/runs/12208665023/job/34062252319?pr=1727